### ISSUE TITLE
It seems your code does not match your description of your workshop paper.

### ISSUE THREAD

**Issue by greathope** [layumi/AICIty-reID-2020#4]
I have read your paper and code roughly, but have few questions.
Your code used SE-ResNeXt101, but ResNeXt101 in your paper; your code did not use Block3; your learning rate is 0.02 instead 0.01 in your paper; the lr scheduler is MultiStepLR  instead Cosine; the feature dim is 4092 instead 1024;  May I ask why?

**Comment by greathope** [comment]
ok, I found it in your paddlepaddle code, so I just close it.

**Comment by layumi** [comment]
Thanks @greathope . We have tried different training methods and then combined the trained model.  In the paper, we mainly report the method to train the best single model (based on validation score).
